mesonbuild/compilers/objcpp.py

```diff
@@ -96,7 +96,9 @@ def get_options(self) -> 'coredata.MutableKeyedOptionDictType':
         opts.update({
             OptionKey('std', machine=self.for_machine, lang='cpp'): coredata.UserComboOption(
                 'C++ language standard to use',
-                ['none', 'c++98', 'c++11', 'c++14', 'c++17', 'gnu++98', 'gnu++11', 'gnu++14', 'gnu++17'],
+                ['none', 'c++98', 'c++11', 'c++14', 'c++17', 'c++20', 'c++2b',
```

Comment:
Do we need version checks on these newer standards?

Reply:
Worth noting that the `darwintests.py` unit test spec currently only tests `c99` for objective-c and `c++14` for objective-c++. (So it currently doesn't even test `c11`, `c17`, `c++17`, etc.)